frontend/src/components/tabs/StaticNodeDetailsV2.tsx

```diff
@@ -199,7 +199,7 @@ function ArtifactNodeDetail({ pipelineSpec, element, layers }: ArtifactNodeDetai
   );
 }
 
-function getComponentSpec(pipelineSpec: PipelineSpec, layers: string[], taskKey: string) {
+export function getComponentSpec(pipelineSpec: PipelineSpec, layers: string[], taskKey: string) {
```

Comment:
Consider moving this function to `NodeUtils.tsx`(new file) or `WorkflowUtils.tsx` since it is exported.